ocaml/doc/index.html

```diff
@@ -78,11 +53,9 @@ <h1 class="title">XenAPI Classes</h1>
 					<ul>
 					<li> set_X: change the value of field X (only if it is read-write);</li>
 					<li> get_X: retrieve the value of field X;</li>
-					<li> add_to_X: add a key/value pair (only if field has type set or map); and</li>
-					<li> remove_from_X: remove a key (only if a field has type set or map).</li>
+					<li> add_to_X: add a key/value pair (only if field has type map); and</li>
+					<li> remove_from_X: remove a key (only if a field has type map).</li>
```

Comment:
Oh, if you fix that here in this PR, then please also add `add_X` and `remove_x` for the set types... :)